www/front_src/src/Resources/Listing/columns/State/DetailsTable/index.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-array-index-key */
```

Comment:
Why this one ?

Reply:
Because there is an index used for a key